src/main/graphql/AddAVMetadata.graphql

```diff
@@ -0,0 +1,11 @@
+mutation AddAVMetadata($input: [AddAVMetadataInput!]!) {
+    addAVMetadata(addAVMetadataInput: $input) {
+        fileId
+        software
+        value
+        softwareVersion
+        databaseVersion
```

Comment:
Did we decide all these fields were necessary? I think Sam knew about what fields were appropriate for YARA.

Reply:
I would rather leave all fields in as it is still vague what is required for yara. If when yara is set up its found that some fields are not needed they can be removed.

Reply:
OK, so we'll come back to it when we do https://national-archives.atlassian.net/browse/TDR-129?